server/flask_server.py

```diff
@@ -0,0 +1,60 @@
+'''
+ RasPiにWebサーバを設置する 
+ @auther  desty505 
+'''
+import os
+import csv
+import re
+from flask import Flask, url_for, request
+
+app = Flask(__name__)
+
+
+# ルーティングを設定
+@app.route('/', methods=["POST", "GET"])
```

Comment:
README見た感じ、データ送信はPOSTで、データ受信はGETでやってるんだよね？
もしそうなら、if文でリクエスト内のテキストの有無を判定するより、GETかPOSTで判断する方がWebらしいと思う(関数名はテキトー)。
参考：https://shigeblog221.com/flask-get-post/

```suggestion
@app.route('/', methods=["GET"])
def read():
    # ファイル読込の処理

@app.route('/', methods=["POST"])
def write():
    # ファイル更新の処理
```

Reply:
分かりました。
ありがとうございます。
参考にさせていただきます。